packages/genesys-spark-components/src/components/stable/gux-icon/gux-icon.types.ts

```diff
@@ -183,9 +183,9 @@ export type GuxIconIconName =
   | 'fa/align-right-regular'
   | 'fa/arrow-down-left-and-arrow-up-right-to-center-regular'
   | 'fa/arrow-down-wide-short-regular'
-  | 'fa/arrow-left-to-line-solid'
+  | 'fa/arrow-left-to-line-regular'
   | 'fa/arrow-right-to-bracket-regular'
-  | 'fa/arrow-right-to-line-solid'
+  | 'fa/arrow-right-to-line-regular'
```

Comment:
We will have to update the script that generates this type. It should include:

- 'fa/arrow-left-to-line-solid'
- 'fa/arrow-right-to-line-solid'
- 'legacy/fa/arrow-left-to-line-solid'
- 'legacy/fa/arrow-right-to-line-solid'

Reply:
done

Reply:
'fa/arrow-left-to-line-solid'
'fa/arrow-right-to-line-solid'
are still missing.

Reply:
Fixed